docs/runtime_api.md

```diff
@@ -314,6 +315,34 @@ JSON.
 To learn about serialization options and other details related to serialization, 
 see the section about [advanced serialization](#advanced-serialization).
 
+### Identifying messages
+
+To check whether a given object is any subtype of `Message` or is a specific
+type of `Message`, we recommend using the exported function `isMessage`.
```

Comment:
Just to shorten it a bit:

```suggestion
To check whether a given object is a message, use the function `isMessage`.
```

Can you link `isMessage` to the source, same as others?